LICENSE.md

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2015
```

Comment:
I don't believe this needs to be updated but not quite sure

Reply:
👍